components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -30,21 +30,37 @@ import {
   RemoveTeamUsers,
   UpdateTeam
 } from 'app/entities/teams/team.actions';
-import { ProjectConstants } from 'app/entities/projects/project.model';
+import { GetProject } from 'app/entities/projects/project.actions';
+import {
+  getStatus as getProjectStatus,
+  projectEntities
+} from 'app/entities/projects/project.selectors';
+import {
+  ProjectChecked
+} from 'app/components/projects-dropdown/projects-dropdown.component';
+
+import { ProjectConstants, Project } from 'app/entities/projects/project.model';
+import { assignableProjects } from 'app/services/projects-filter/projects-filter.selectors';
+import { ProjectsFilterOption } from 'app/services/projects-filter/projects-filter.reducer';
+import { IAMType } from 'app/entities/policies/policy.model';
 
 const TEAM_DETAILS_ROUTE = /^\/settings\/teams/;
 
 export type TeamTabName = 'users' | 'details';
 
+export interface ProjectMap {
+  [id: string]: ProjectChecked;
+}
```

Comment:
Ah, looks like should move this `ProjectMap` definition to projects-dropdown.component.ts so that it can be shared. Should also be used in the definition of `projects` in create-object-modal.component.ts (L30).